backends-clickhouse/src/test/scala/io/glutenproject/benchmarks/DSV2BenchmarkTest.scala

```diff
@@ -320,9 +320,7 @@ object DSV2BenchmarkTest extends AdaptiveSparkPlanHelper {
                               |
                               |""".stripMargin) // .show(30, false)
         // df.queryExecution.debug.codegen
-        // df.explain(false)
```

Comment:
don't need modify this file and `[DSV2TPCDSBenchmarkTest.scala](https://github.com/oap-project/gluten/pull/611/files#diff-7fd63601b08c75a30713b37065381fb835cba83a81a6336e7900ece1e8f9ebd5)`, I have removed these two files in another PR.